cloud_sub_topic: /kitti/velo/pointcloud
imu_sub_topic: /kitti/oxts/imu
velocity_sub_topic: /kitti/oxts/gps/vel
gnss_sub_topic: /kitti/oxts/gps/fix


calibration_param:
  lidar_to_imu: [0.999998  ,    0.000755307 ,  -0.00203583 ,   -0.808676,
 -0.000785403 ,     0.99989 , -0.014823 ,    0.319556,
 0.00202441 ,   0.0148245 ,    0.999888 ,   -0.799723,
 0 ,           0 ,           0,            1]
    
